src/java/org/apache/cassandra/service/CassandraDaemon.java

```diff
@@ -316,6 +317,11 @@ private void recoverCommitlogAndCompleteSetup() {
         {
             StorageService.instance.initServer();
         }
+        catch (BootstrappingSafetyException e)
+        {
+            System.out.println(e.getMessage() + "\nNon-fatal bootstrap error. Server will continue but is disabled and without metrics.");
```

Comment:
Why are we using System.out instead of the `logger` (here as well as in the pre-existing catch-es)? @leonz will we get these in Aries?

Reply:
No, we shouldn't use System.out

Reply:
I'll refactor this and the chained catches within the function. There are a few points in the o.a.c.service.CassandraDaemon that use System.out instead of the logger but these should be refactored separately.